cli/flox/src/commands/services/mod.rs

```diff
@@ -206,21 +233,42 @@ pub fn warn_manifest_changes_for_services(flox: &Flox, env: &dyn Environment) {
 /// If an invalid name is provided, an error is returned.
 fn processes_by_name_or_default_to_all<'a>(
     processes: &'a ProcessStates,
+    manifest_services: &ManifestServices,
```

Comment:
nit: it was a bit confusing for me that "all" means "all on this system because processes will be restricted to this system." Not sure if that's worth a comment